documents/coredocument.go

```diff
@@ -441,6 +441,7 @@ func (cd *CoreDocument) createProofs(fromZKTree bool, docType string, dataLeaves
 
 	return &DocumentProof{
 		FieldProofs:    rawProofs,
+		DataRoot:       targetTree.RootHash(),
```

Comment:
Since the inner data trees are static, maybe we can rename this as well to:
- `DataRoot` to `BasicDataRoot` or `LeftDataRoot`
- `SiblingRoot` to `ZkDataRoot` or `RightDataRoot`
So any consumer of this can ultimately know what to do with this response.

Reply:
I would like to keep this as is because, target tree can be zk tree or basic treee. It would bee confusing to put either because we might give out wrong naming when the flag for zk is turned on. WDYT?

Reply:
The problem I see is that since `targetTree` and `siblingTree` will be swaped (in content) then when enabling zk flag, NFT logic will fail since it expects static_hashes={basicroot,zkroot,signaturesroot}, then the centchain nft module will consume the wrong order for dataRoot and siblingRoot.